govcd/vdccomputepolicy_v2.go

```diff
@@ -182,28 +182,40 @@ func (vdcComputePolicy *VdcComputePolicyV2) Delete() error {
 // GetAllAssignedVdcComputePoliciesV2 retrieves all VDC assigned Compute Policies (V2) using OpenAPI endpoint. Query parameters can be supplied to perform additional
 // filtering
 func (vdc *AdminVdc) GetAllAssignedVdcComputePoliciesV2(queryParameters url.Values) ([]*VdcComputePolicyV2, error) {
+	return getAllAssignedVdcComputePoliciesV2(vdc.client, vdc.AdminVdc.ID, queryParameters)
+}
+
+// GetAllAssignedVdcComputePoliciesV2 retrieves all VDC assigned Compute Policies (V2) using OpenAPI endpoint. Query parameters can be supplied to perform additional
+// filtering
+func (vcdClient *VCDClient) GetAllAssignedVdcComputePoliciesV2(vdcId string, queryParameters url.Values) ([]*VdcComputePolicyV2, error) {
+	return getAllAssignedVdcComputePoliciesV2(&vcdClient.Client, vdcId, queryParameters)
```

Comment:
For clarity I think it is worth specifying `vdcId` is mandatory and possibly even fails here before calling `getAllAssignedVdcComputePoliciesV2`. It might return a very odd error because of malformed request url

Reply:
Done, thanks for spotting!